packages/curve-ui-kit/src/themes/components/chip/mui-chip.ts

```diff
@@ -69,9 +69,9 @@ export const defineMuiChip = (
         '& .MuiChip-label': {
           paddingInlineStart: 0,
           paddingInlineEnd: Spacing.xs.desktop,
-          '&:empty': { display: 'none' },
```

Comment:
Note: This was only working when using Chip with MUI icons (due to `.MuiChip-icon` check), but we can use Chip with any image